docs/GettingStartedGuide.md

```diff
@@ -1,19 +1,57 @@
 # Getting Started Guide
 
-To begin with `harp.gl`, we provide few starting points
+To begin with `harp.gl`, we provide a few starting points:
 
--   [Create simple app](#yeoman) using Yeoman
--   [Integrate `harp.gl` into your existing Webpack based project](#integrate)
--   [Look at examples](#examples)
+- [Import harp.gl with simple bundle](#simple)
+- [Create simple app](#yeoman) using Yeoman
+- [Integrate `harp.gl` into your existing Webpack based project](#integrate)
+- [Look at examples](#examples)
+
+## <a name="simple"></a> Import harp.gl with simple bundle
```

Comment:
Can this instead not link back to the README.md examples? Having two versions increases the change that they differentiate and / or maintenance cost.

Reply:
This examples link is from a prior commit. However, I don’t understand your question. Where would you like this to link to? And where is the duplication?

Reply:
Sorry I wasn't clear, I mean the html and javascript below are exactly the same in both places.

Reply:
Ah, I understand now. I do think it's crucial to have basic getting started information on the repo's top-level readme file (hence this pull request) in order to make it as easy as possible for new users to get started with harp.gl. But yes, this information should also be included in the getting started guide page for obvious reasons.

I would prioritize having this getting started information in two visible locations (top-level readme and getting started readme) rather than getting started only, even if that means having two versions with the risk of them possibly getting out of sync. At some point, having a single source of code samples will be unavoidable as the project grows. 

What does the rest of the team think?

Reply:
Valid points!